RenderSystems/Vulkan/CMakeLists.txt

```diff
@@ -28,10 +28,14 @@ endif()
 target_include_directories(RenderSystem_Vulkan PUBLIC
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
     $<INSTALL_INTERFACE:include/OGRE/RenderSystems/Vulkan>)
+    
+if(WIN32)
+    target_include_directories(RenderSystem_Vulkan PUBLIC $ENV{VULKAN_SDK}/Include)
```

Comment:
are you running into problems with a static build, or why do you need the headers?